server/asset/dcr/dcr.go

```diff
@@ -155,6 +165,15 @@ func (dcr *DCRBackend) CheckAddress(addr string) bool {
 	return err == nil
 }
 
+func (dcr *DCRBackend) UnspentCoinDetails(coinID []byte) (addr string, val uint64, confs int64, err error) {
```

Comment:
Unless you object, I think it would be good to keep the UTXO version available although obv not part of the DEXAsset interface.  ba76c77 docs UnspentCoinDetails, tests it directly, and renames UnspentDetails to UTXODetails.